src/relay/op/tensor/tensor.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ * \file tensor.h
+ * \brief Declarations for tensor operations.
+ */
+#ifndef TVM_RELAY_OP_TENSOR_TENSOR_H_
+#define TVM_RELAY_OP_TENSOR_TENSOR_H_
+
+#include "tvm/relay/expr.h"
+
+namespace tvm {
+namespace relay {
+
+Expr Prod(Expr data, Array<Integer> axis, bool keepdims, bool exclude);
```

Comment:
BTW, we already have a bunch of op definitions in 'pattern_utils.h'. We can probably move this one there as well.

Reply:
Thanks, didn't notice reduce ops are also defined there.

Reply:
But I do find that it is a bit weird to declare those make-op in `patterns_utils.`. `tensor.h` would be more appropriate, I think.